tensorflow/core/profiler/internal/tfprof_constants.h

```diff
@@ -31,6 +31,9 @@ static const char* const kTrainableVarType = "_trainable_variables";
 // Op type for tensors in the checkpoint file.
 static const char* const kCkptVarType = "_checkpoint_variables";
 
+static const char* const kDeviceManufacturerNvidia = "Nvidia";
```

Comment:
Please move these constants to, e.g., device_caps_utils.h.

tfprof_constants is part of now deprecated TF1 profiler infra.

Reply:
Done in  f9717ce.